components/VotePanel/YouVoted.tsx

```diff
@@ -180,19 +199,37 @@ export const YouVoted = ({ quorum }: { quorum: 'electoral' | 'veto' }) => {
       {(isVoting || isInCoolOffTime) && (
         <div className="items-center justify-center flex w-full gap-5">
           <div className="flex flex-col gap-6 items-center">
-            <SecondaryButton
-              className="min-w-[200px]"
-              isLoading={isLoading}
-              tooltipMessage={withdrawTooltipContent}
-              onClick={() => submitRelinquishVote()}
-              disabled={!isWithdrawEnabled || isLoading}
-            >
-              Relinquish Vote
-            </SecondaryButton>
+            {isMulti ?
+              <Button
+                className="min-w-[200px]"
+                isLoading={isLoading}
+                tooltipMessage={withdrawTooltipContent}
+                onClick={() => submitRelinquishVote()}
+                disabled={!isWithdrawEnabled || isLoading}
+              >
+                Change Vote
```

Comment:
see previous comment, just make them both say the same thing. "Withdraw Vote" is probably more clear than either "Change Vote" or "Relinquish Vote"